Apps/AdminWebClient/src/ClientApp/src/components/core/modals/EmailModal.vue

```diff
@@ -17,7 +17,7 @@
                             <v-text-field
                                 v-model="editedItem.subject"
                                 label="Subject"
-                                :rules="[v => !!v || 'Subject is required']"
+                                :rules="[(v) => !!v || 'Subject is required']"
```

Comment:
shouldn't the local lint have catch this?

Reply:
Yes.  I did some digging and there seems to be something wrong with the prettier formatting in the AdminWebClient ClientApp.  It removes braces from inline functions as seen above.  Also, it removes the trailing comma on the last item in imports and array/object declarations and doesn't format those lists properly if you mess with the whitespace/newlines.

I wasn't able to figure out a way to resolve the underlying issue, but if you run prettier from a directory higher than ClientApp, it will apply the appropriate formatting. (e.g. `npx prettier -w . '!**/{bin,obj,dist,sonar_reports}/**'`)

However, if you run the linter in the project or save one of the affected files in VSCode (with formatOnSave enabled), the bad formatting will be reinstated.